src/Core/View.js

```diff
@@ -1057,9 +1057,9 @@ class View extends THREE.EventDispatcher {
      * @param   {Coordinates}           [target]    The result will be copied into this {@link Coordinates}. If not
                                                     * specified, a new {@link Coordinates} instance will be created.
      *
-     * @returns {Coordinates}   The world {@link Coordinates} at the given view coordinates.
+     * @returns {Coordinates}   The world {@link Coordinates} of the terrain at the given view coordinates.
      */
-    pickCoordinates(mouse, target = new Coordinates(this.tileLayer.extent.crs)) {
+    pickTerrainCoordinates(mouse, target = new Coordinates(this.referenceCrs)) {
```

Comment:
I think yes because it is clearer for users since in my opinion users don't really know the crs of the extent of the tileLayer which is quite internal to itowns, they should expect to get coordinates in the view CRS which they actually know of. You're right that it is a breaking change, I will add one.